main.py

```diff
@@ -50,30 +57,34 @@ def draw_bars(self, year, month):
         raw_data = self.file_reader.read_weather_file(file)
         weather_data = [WeatherData.from_csv_row(row) for row in raw_data]
 
-        WeatherStats.find_daily_extremes(weather_data)
+        for data in weather_data:
+            if data.max_temp is not None and data.min_temp is not None:
+                print(
+                    f"{data.date} {Fore.BLUE + '+' * data.min_temp + Fore.RESET + Fore.RED + '+' * data.max_temp + Fore.RESET} {data.min_temp}C - {data.max_temp}C"
```

Comment:
It looks a little confusing. Instead of using single print use multiple prints and add comments with them.